### ISSUE TITLE
AtAddress button is still clickable when it's disabled 

### ISSUE THREAD

**Issue by drafish** [ethereum/remix-project#3889]
AtAddress button is still clickable when it's disabled. And when it's enabled, cursor does not become small hand, it's still an arrow.

I know it's a small problem. But I still hope remix can fix it.

I'v already fixed it in brochain ide. I will submit a pr for remix.

**Comment by drafish** [comment]
@Aniket-Engg It is fixed now. But it's not perfect. When `AtAddress` is enabled, cursor does not become a small hand, it's still an arrow. I mean it's a button, right, cursor should become a small hand when I hover on the button.

I see the way that @LianaHus fixed this issue.

```javascript
if (address == '') return
```

To my understanding, it's still clickable. It's just be returned when `address == ''`, so that tooltip wouldn't show up. But actually the click event is still be triggered.

Here is how I fixed this issue in brochain ide.
![image](https://github.com/ethereum/remix-project/assets/18543527/41d59528-39a4-4fe1-a415-18aa305a9734)

I removed `onClick={loadFromAddress}` on div element, because button element already has `onClick`. And I removed `style={{ pointerEvents: 'none' }}` on button element, so that button's `onClick` will work.

I think it will be better if `onClick` is triggered by button element instead of div element. Whether or not the button is disabled is controlled by `disabled={atAddressOptions.disabled}`. If the button is disabled, `onClick` will not be triggered, and `if (address == '') return` will not be necessary. And the cursor issue will be fixed too.

I don't understand why there is `onClick` on div element, when button already has `onClick`. Is there any other concerns?
